doc/manpages/man1/qrest.fr.1

```diff
@@ -0,0 +1,30 @@
+.TH man 1 "12 Jan 2013" "0.5" "Qrest"
+.SH NOM
+qrest \- Qrest Really Easy Studio Toolkit
+.SH SYNOPSIS
+qrest
+.SH DESCRIPTION
+.B qrest
+aide à calculer les temps de délai et les fréquences de LFO en fonction du tempo.
+.P
+Il y a 2 façons d'entrer les valeurs de tempo :
+.IP \(bu 2
+Entrée numérique au clavier : l'utilisateur entre simplement la valeur de tempo dans un champ de texte et appuie sur \'entrée\'.
+.IP \(bu 2
+Clic au bouton en temps-réel : l'utilisateur clique sur un bouton en synchronisation avec des pulsations, et qrest lui dira s'il tape régulièrement.
```

Comment:
'avec des pulsations' -> traduction de 'in sync with beats'. Tu préfèrerais "rythmes" ?

'qrest lui dira' -> c'est la simple traduction de "and qrest will tell her if she taps steadily.", si t'as une meilleure traduction, dis moi. "Réveler" ? Ça me semble un peu lourdeau dans la phrase.